### PR TITLE
sci-libs/HighFive: 2.8.0 version bump

### PR DESCRIPTION
**Pull request by band-a-prend**

### CODE REVIEW

File: sci-libs/HighFive/HighFive-2.8.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+KEYWORDS="~amd64 ~x86"
+LICENSE="Boost-1.0"
+SLOT="0"
+IUSE="mpi test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	sci-libs/hdf5[mpi?]
+	doc? ( app-doc/doxygen[dot] )
+
```
Comment (mm1ke) [inline sci-libs/HighFive/HighFive-2.8.0.ebuild:24]: guess this empty line could be removed

Comment (band-a-prend) [reply sci-libs/HighFive/HighFive-2.8.0.ebuild]: Thanks for the notice! Done.

File: sci-libs/HighFive/HighFive-2.8.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+KEYWORDS="~amd64 ~x86"
+LICENSE="Boost-1.0"
+SLOT="0"
+IUSE="mpi test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	sci-libs/hdf5[mpi?]
+	doc? ( app-doc/doxygen[dot] )
```
Comment (AndrewAmmerlaan) [inline sci-libs/HighFive/HighFive-2.8.0.ebuild:23]: This is probably not correct, I doubt we need doxygen at runtime.  Also you can use `DOCS_DEPEND` to specify this dependency.

Comment (band-a-prend) [reply sci-libs/HighFive/HighFive-2.8.0.ebuild]: Great thanks for review. I completely forgot about `DOCS_DEPEND` feature.
The `app-text/doxygen[dot]` dependency is moved to this variable now.

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**